src/Uno.UI/UI/Xaml/Controls/WebView/Native/iOSmacOS/UnoWKWebView.iOSmacOS.cs

```diff
@@ -67,7 +67,7 @@ public partial class UnoWKWebView : WKWebView, INativeWebView, IWKScriptMessageH
 			}
 		}
 
-#if __IOS__ && DEBUG
+#if __IOS__
```

Comment:
I thought we wanted to get this available only when `DEBUG`

Reply:
Right, we do not provide debug builds of our nuget packages. The docs should only mention this flag as debug.

Reply:
Right makes sense.